### PR TITLE
Mat 5470 bundle null nqf

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-5470](https://jira.cms.gov/browse/MAT-5470)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by nmorasb — DISMISSED** [review]

File: src/main/java/gov/cms/madie/madiefhirservice/services/MeasureTranslatorService.java [hunk 1]
```diff
@@ -127,15 +127,19 @@ public List<Identifier> buildMeasureIdentifiers(Measure madieMeasure) {
               ValueConstants.CODE_VERSION_SPECIFIC));
       if (madieMeasure.getMeasureMetaData() != null
           && madieMeasure.getMeasureMetaData().getEndorsements() != null
-          && !madieMeasure.getMeasureMetaData().getEndorsements().isEmpty()) {
+          && !madieMeasure.getMeasureMetaData().getEndorsements().isEmpty()
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/madiefhirservice/services/MeasureTranslatorService.java]: Consider combining these two checks with `CollectionUtils.isEmpty()`

**Review by jkotanchik-SB — DISMISSED** [review]

File: src/main/java/gov/cms/madie/madiefhirservice/services/MeasureTranslatorService.java (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/madiefhirservice/services/MeasureTranslatorService.java]: @jkotanchik-SB Just made the change, thanks!

**Review by jkotanchik-SB — APPROVED** [review]
